### ISSUE TITLE
Deduplicate "Can be cut down with the right tools" in description of trees

### ISSUE THREAD

**Issue by NetSysFire** [CleverRaven/Cataclysm-DDA#55207]
### Is your feature request related to a problem? Please describe.

It is annoying to have *every* tree description say "It can be cut down with the right tools." at the end.

### Solution you would like.

Chopping down a tree works via the TREE flag. If the flag is present, farlooking the tree should append a "It can be <green>cut down</color> with the right tools" like you have with items.

This *may* be a good first issue but I do not know enough about the internals to say that for sure. Since this is not an item but rather something you inspect from a distance, this may be a bit more tricky.

**Comment by LeahLuong** [comment]
Prob wouldn't hurt to simply remove it from the description. It's not like felling isn't 1 of the oldest activities known to humanity. It's also been a feature of sandbox video games since pterodactyls flew the skies.

**Comment by NetSysFire** [comment]
Some objects already have info such as this:

![image](https://user-images.githubusercontent.com/59517351/153567106-ffe5f278-398b-4f0c-bb5b-400558544615.png)

I would put it on its own line with the other information. See what looks best, it might be good to place it above the "Cover" and other properties.

**Comment by haveric** [comment]
I've got a couple options that can be implemented. This has both the description and a `Cuttable` tag added.
![image](https://user-images.githubusercontent.com/985060/153651447-731e2652-7b2d-40df-8da5-f8a1dbb023d0.png)


Alternative positioning with modified text:
![image](https://user-images.githubusercontent.com/985060/153651317-6d53f8f5-37f3-4b96-be20-11a1144c5cc4.png)

**Comment by NetSysFire** [comment]
Nice! Regardless of the actual choice, please color "cut down" green. Right now, I would go for the second one since afaik all the properties of items and terrain are described more neutral and not in first person. Thanks to your POC screenshots I can isolate the issue better and it looks more "natural" when placed with the other properties.

The only other thing I could imagine right now is the duplication in "Cuttable" since "Can be cut down with the right tools" already covers that. Or vice-versa, but this becomes bike-shedding now.

**Comment by haveric** [comment]
Yeah, I threw both the text and the `Cuttable` into the same screenshots to avoid needing to make more screenshots. I think I'm going to remove the `Cuttable`, but it'll take me a bit to figure out how to colorize `cut down` correctly in this context.

**Comment by LeahLuong** [comment]
There should be a "the" preceding "New England" & a comma after "closely" in the 2nd sentence.

**Comment by haveric** [comment]
> There should be a "the" preceding "New England" & a comma after "closely" in the 2nd sentence.

That's unrelated to this issue. There's likely more issues like this across all of the trees.

**Comment by NetSysFire** [comment]
Yeah, this is not a "technical" issue relating to code, just manual and boring grammar checking and therefore out of scope here.

Just creating an issue for each case of faulty grammar you find is not helpful though since fixing it is faster than reporting it. If you are looking for a good first thing to start contributing to, you just found it.

**Comment by LeahLuong** [comment]
Well, excuse me, I simply thought if they were going to be editing the text in this file it wouldn't be a big deal to correct the  grammar errors @ the same time.

**Comment by haveric** [comment]
> Well, excuse me, I simply thought if they were going to be editing the text in this file it wouldn't be a big deal to correct the grammar errors @ the same time.

I'm not sure why the attitude is needed as we're just trying to be helpful here. While it is easy to edit a little bit of text while making this change, it then becomes more difficult for a reviewer to review it as it's adding more complexity to the changes, especially as you would likely need to edit more than just this one tree to fix all of the issues with grammar related to this change.

TLDR: Because I edited the descriptions of all trees, I wouldn't be doing a grammar edit justice by only editing this one without spending the extra time needed to check the rest.
